spec/rubocop/cop/ezcater/rspec_require_feature_flag_mock_spec.rb

```diff
@@ -0,0 +1,35 @@
+require "spec_helper"
+
+RSpec.describe RuboCop::Cop::Ezcater::RspecRequireFeatureFlagMock, :config do
```

Comment:
Aside: how do you feel about requiring the `RSpec.` here?

Reply:
for some reason i thought rspec recommended one over the other? cant remember tho

Reply:
I like this - we do this in sem-tools as well. You also made it part of the configuration in the gem, so I assumed you desired this as well. If not, we would need to remove `config.disable_monkey_patching!` from `spec_helper.rb`

Reply:
When you run `rspec install` now it includes the `disable_monkey_patching!` so I left it as is. 

Since it seems to be their recommendation, and it's a pretty minimal change, just being on the `describe`, I'm good with it. 👍